Makefile

```diff
@@ -45,11 +45,13 @@ CHART_PATH ?= kubernetes
 CHART_NAME_1 ?= cray-service
 CHART_NAME_2 ?= cray-jobs
 CHART_NAME_3 ?= cray-postgresql
-CHART_VERSION_1 ?= local
-CHART_VERSION_2 ?= local
-CHART_VERSION_3 ?= local
+CHART_VERSION_1 ?= 0.0.0
+CHART_VERSION_2 ?= 0.0.0
+CHART_VERSION_3 ?= 0.0.0
```

Comment:
Make it possible to run tests locally by just running `make charts`. When using `local` as a version default, this errors out caliming `local` is not a valid SemVer.